packages/block-library/src/post-title/block.json

```diff
@@ -34,7 +34,11 @@
 		"html": false,
 		"color": {
 			"gradients": true,
-			"link": true
+			"link": true,
+			"__experimentalDefaultControls": {
```

Comment:
This block has a "Make title a link" setting. Should we add link as one of the defaults?